### PR TITLE
dash/oren/v2-poa-requests/oas

### PR DESCRIPTION
**Pull request by nihil2501**
To get a nice web view of proposed schema and corresponding examples, download, extract, and open this in your browser: 
[v2-poa-requests.zip](https://github.com/user-attachments/files/15875070/v2-poa-requests.zip)

### CODE REVIEW

File: modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json [hunk 1]
```diff
@@ -69,6 +69,1310 @@
...
+        "summary": "Create the decision for Power of Attorney requests.",
+        "tags": [
+          "Power of Attorney"
+        ],
+        "operationId": "createPowerOfAttorneyRequestDecisions",
+        "security": [
+          {
+            "productionOauth": [
+              "system/claim.read",
+              "system/system/claim.write"
```
Comment (FonzMP) [inline modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json:83]: typo? it's in multiple spots as well - I know I saw some of this in another section of the documentation so assuming it was carried over.

File: modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json [hunk 2]
```diff
@@ -69,6 +69,1310 @@
...
+            ]
+          },
+          {
+            "sandboxOauth": [
+              "system/claim.read",
+              "system/system/claim.write"
```
Comment (FonzMP) [inline modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json:89]: again, just `system/` not `system/system/`

File: modules/claims_api/app/controllers/claims_api/v2/power_of_attorney_requests/decisions_controller.rb [hunk 3]
```diff
@@ -12,7 +12,7 @@ def create
             deserialize(@body)
           )
 
-          head :no_content
+          render json: {}, status: :accepted
```
Comment (FonzMP) [inline modules/claims_api/app/controllers/claims_api/v2/power_of_attorney_requests/decisions_controller.rb:15]: Is there a ticket for defining the 202 response? I believe in the other 202's we return roughly the data they sent in the request just as an acknowledgement of "hey, we got this data"... trying to locate if that was due to the Lighthouse standards, but just curious where we stood. Can be done in a separate ticket if we want to continue to move on this one and circle back.

Comment (rockwellwindsor-va) [reply modules/claims_api/app/controllers/claims_api/v2/power_of_attorney_requests/decisions_controller.rb:15]: Was wondering also if there should be a schema in the docs for that 202?  For other 202s we do have the schema in there but not sure what was all discussed for this one.

For example:
/veteran/{id}/526
Screenshot 2024-06-18 at 1 18 32 PM

.../decision
Screenshot 2024-06-18 at 1 18 20 PM

Comment (nihil2501) [reply modules/claims_api/app/controllers/claims_api/v2/power_of_attorney_requests/decisions_controller.rb:15]: @FonzMP Yeah my plan was to revisit the spec for the POST decision response after implementing the auto-establishment piece of that call.

File: modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json (hunk 2 above)
Comment (nihil2501) [reply modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json:89]: whoops, yeah I copy pasta'd from somewhere.

**Review by FonzMP — APPROVED** [review]
:shipit:

**Pull request merged**